scss/2-tools/_grid.scss

```diff
@@ -0,0 +1,13 @@
+@use 'sass:math';
```

Comment:
As nice as the ITCSS structure is generally I don't _love_ having the settings / helpers / and styles for grids in three different files. Following the existing path here though.